examples/llm/src/models/layers/norm.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2022 MosaicML Examples authors
+# SPDX-License-Identifier: Apache-2.0
+import torch
+from composer.algorithms.low_precision_layernorm.low_precision_layernorm import \
```

Comment:
can you copy lpln from composer into examples as part of this pr? it makes it easier to remove the composer dependency for hf release